node/external/transactionAPI/apiTransactionResults.go

```diff
@@ -109,27 +107,29 @@ func (arp *apiTransactionResultsProcessor) putSmartContractResultsInTransactionB
 		}
 
 		scrAPI := arp.adaptSmartContractResult(scrHash, scr)
-		arp.putLogsInSCR(scrHash, epoch, scrAPI)
+		arp.loadLogsIntoContractResults(scrHash, epoch, scrAPI)
 
 		tx.SmartContractResults = append(tx.SmartContractResults, scrAPI)
 	}
 }
 
-func (arp *apiTransactionResultsProcessor) putLogsInTransaction(hash []byte, tx *transaction.ApiTransactionResult, epoch uint32) {
+func (arp *apiTransactionResultsProcessor) loadLogsIntoTransaction(hash []byte, tx *transaction.ApiTransactionResult, epoch uint32) {
 	var err error
 
 	tx.Logs, err = arp.logsFacade.GetLog(hash, epoch)
-	if err != nil && !errors.Is(err, storage.ErrKeyNotFound) {
-		log.Warn("putLogsInTransaction()", "hash", hash, "epoch", epoch, "err", err)
+	if err != nil {
+		// TODO: We should unwrap the error, reason about its type and possibly ignore "key not found in storage" errors.
```

Comment:
if those errors are to be expected in some cases, we might print them as TRACE and remove TODOs: valid for all occurrences. Can be done in another PR

Reply:
Removed TODO. Switched to TRACE.